iree/turbine/kernel/ops/wave_ops.py

```diff
@@ -611,7 +615,7 @@ def type(self) -> Memory:
             raise ValueError(
                 "BinaryPyOp requires lhs and rhs shape to be at least broadcastable."
             )
-        broadcasted_type = lhs_type if lhs_dim_set > rhs_dim_set else rhstype
+        broadcasted_type = lhs_type if lhs_dim_set > rhs_dim_set else rhs_type
```

Comment:
Nice! Not sure how that got by code review :)

Reply:
haha yee :)